digiwf-task/digiwf-tasklist-service/src/main/resources/application.yml

```diff
@@ -49,6 +49,11 @@ feign:
       default:
         url: "http://localhost:${ENGINE_SERVER_PORT}/engine-rest/"
 
+easyldap:
+  client:
+    url: ""
```

Comment:
Should reference an environment variable.

Reply:
Changed